_posts/2015-08-13-livechat.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: LiveChat Integration
+categories: integrations
+author: Ryan Salomon
+tags: [livechat, integrations]
+summary: Learn about our LiveChat Integration
+---
+
+* Table of Contents
+{:toc}
+* * *
+
+The LiveChat/Kissmetrics integration gives you the ability to see just how effective LiveChat is at creating conversions and increasing conversion rate. Here’s the list of events we’ll be tracking with this integration: 
+
+LiveChat Automated greeting displayed (greetings can be configured in LiveChat’s Settings > Chat window > Greetings)<br>
+LiveChat Chat started<br>
+LiveChat Pre-chat survey displayed<br>
+LiveChat Pre-chat survey filled in<br>
+LiveChat Post-chat survey displayed<br>
+LiveChat Post-chat survey filled in<br>
+LiveChat Ticket form displayed ("ticket form" is displayed when no chat agents are available to chat)
+LiveChat Ticket form filled in
+
+For simplicity, we’ve given you a property named ‘Performed Chat’. If a user has a value of ‘true’ for this property, you know they’ve performed a chat.  You can then use this property across all of your Kissmetrics reports to see how LiveChat users differ from your other users across all the events you’re already tracking.  
+
+![LiveChat 1][livechat_image_1]<br>
```

Comment:
The trailing `<br>` in this case is probably unnecessary because you've already put this in its own paragraph (`<p>`) by having a line break before and after.